src/main/kotlin/xyz/atrius/waystones/commands/ConfigCommand.kt

```diff
@@ -7,29 +7,35 @@ import xyz.atrius.waystones.utility.message
 object ConfigCommand : SimpleCommand("config", "conf", "co", "c") {
 
     override fun execute(sender: CommandSender, args: Array<String>) {
+        // Permissions Check
+        if (!sender.hasPermission("waystones.config")) {
+            sender.message("&7[&dWaystones&7] &cYou don't have permission to run this!")
+            return
+        }
```

Comment:
This can be simplified to
```suggestion
        if (!sender.hasPermission("waystones.config"))
            return sender.message("&7[&dWaystones&7] &cYou don't have permission to run this!")
```